api/go/info/info.pb.go

```diff
@@ -875,11 +985,11 @@ func (x BootReason) String() string {
 }
 
 func (BootReason) Descriptor() protoreflect.EnumDescriptor {
-	return file_info_info_proto_enumTypes[12].Descriptor()
+	return file_info_info_proto_enumTypes[14].Descriptor()
```

Comment:
These kind of constant value updates scares me, its very easy to miss the updates at few places. Can't we define a global const and use that ? Then the future changes will be in just one place understand that you did not introduce that in this commit, but may be its time to fix that properly. I see majority of the changes in this commit is actually updating the constant values.

Reply:
This file is generated automatically